lib/_stream_pump.js

```diff
@@ -0,0 +1,191 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2014 Mathias Buus
```

Comment:
Is the idea that this file keeps getting vendored in? So I guess review comments on the code here make limited sense?